elastic/kibana
# [Fleet] Unenroll inactive agent when resetting agent policies

## Summary

In 8.13 we introduced a new validation to not delete an agent policy if some agent are still enrolled to that policy.

This cause issue when trying to use the reset cloud agent policy API.

That PR fix that by unenrolling inactive agent when reseting a preconfigured API.


@criamico @juliaElastic I am wondering if when passing the `force` flag to agent policy delete service we should skip the agent validation.

## How to tests

Create a preconfigured agent policy, add some agent, and check you reset the agent policy.
